src/VisualStudio/Core/Def/Implementation/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -170,8 +168,9 @@ private AbstractTableDataSourceFindUsagesContext StartSearchWorker(string title,
         {
             this.AssertIsForeground();
 
+            var vsFindAllReferencesService = (IFindAllReferencesService)_serviceProvider.GetService(typeof(SVsFindAllReferences));
```

Comment:
Now that this is here, you might want to explicitly in the base constructor call to write assertIsForeground: false. It is the default, but now we can document the actual intent here.

Reply:
The explicit parameter isn't necessary IMO. The options are passing the default value false, and causing a bug by passing true. 😄